_docs/10-support.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: splash
+title: "Support"
+permalink: /docs/support/
+hidden: true
+header:
+  overlay_color: "#5e616c"
+  overlay_image: /assets/images/201106-CORP-BNR-Design-Centers-FPGAs-and-plds-Banner-2880x280.jpg
+excerpt: "Welcome to the Support page."
+---
+
+For general Libero SoC issues please contact [Microchip Technical Support](http://soc.microsemi.com/Portal/DPortal.aspx?v=1).
```

Comment:
Is this still the correct support link for Libero? i.e. the Microsemi portal, rather than Microchip tech support? KK question?

Reply:
yes this is the link that I got from KK when this page was initially created. 
Yes that one link would apply to all. Happy to change it to an alterative.